resources/l10n/de/LC_MESSAGES/loot.po

```diff
@@ -1,584 +1,742 @@
 msgid "Total Plugins"
 msgstr "Plugins insgesamt"
 
-#: src/gui/html/js/translateStaticText.js:247
+#: src/gui/html/js/translateStaticText.js:525
 msgid "Default Game"
 msgstr "Standard-Spiel"
 
-#: src/gui/html/js/translateStaticText.js:248
+#: src/gui/html/js/translateStaticText.js:528
 msgid "Autodetect"
 msgstr "Automatisch erkennen"
 
-#: src/gui/html/js/translateStaticText.js:253
+#: src/gui/html/js/translateStaticText.js:539
 msgid "Language changes will be applied after LOOT is restarted."
 msgstr "Spracheinstellungen werden wirksam nachdem LOOT neugestartet wird."
 
-#: src/gui/html/js/translateStaticText.js:255
+#: src/gui/html/js/translateStaticText.js:544
 msgid "Enable debug logging"
 msgstr "Aktivere Debug-Logging"
 
-#: src/gui/html/js/translateStaticText.js:256
+#: src/gui/html/js/translateStaticText.js:548
 msgid "The output is logged to the LOOTDebugLog.txt file."
 msgstr "Die Ausgabe wird in die LOOTDebugLog.txt-Datei geloggt."
 
-#: src/gui/html/js/translateStaticText.js:258
+#: src/gui/html/js/translateStaticText.js:554
 msgid "Update masterlist before sorting"
 msgstr "Aktualisiere Masterlist vor dem Sortieren"
 
-#: src/gui/html/js/translateStaticText.js:262
+#: src/gui/html/js/translateStaticText.js:560
+msgid "Check for LOOT updates on startup"
+msgstr "Prüfe auf LOOT Updates beim Start"
+
+#: src/gui/html/js/translateStaticText.js:568
 msgid "Base Game"
 msgstr "Grund-Spiel"
 
-#: src/gui/html/js/translateStaticText.js:263
+#: src/gui/html/js/translateStaticText.js:571
 msgid "LOOT Folder"
 msgstr "LOOT Ordner"
 
-#: src/gui/html/js/translateStaticText.js:265
+#: src/gui/html/js/translateStaticText.js:577
 msgid "Masterlist Repository URL"
 msgstr "Masterlist Repository URL"
 
-#: src/gui/html/js/translateStaticText.js:266
+#: src/gui/html/js/translateStaticText.js:580
 msgid "Masterlist Repository Branch"
 msgstr "Masterlist Repository Branch"
 
-#: src/gui/html/js/translateStaticText.js:267
+#: src/gui/html/js/translateStaticText.js:583
 msgid "Install Path"
 msgstr "Installationspfad"
 
-#: src/gui/html/js/translateStaticText.js:268
+#: src/gui/html/js/translateStaticText.js:586
 msgid "Install Path Registry Key"
 msgstr "Installationspfad Registrierungsschlüssel"
 
-#: src/gui/html/js/translateStaticText.js:281
+#: src/gui/html/js/translateStaticText.js:606
 msgid "First-Time Tips"
 msgstr "Anfänger-Tips"
 
-#: src/gui/html/js/translateStaticText.js:283
+#: src/gui/html/js/translateStaticText.js:609
+#, javascript-format
+msgid ""
+"This appears to be the first time you have run LOOT v%s. Here are some tips "
+"to help you get started with the interface."
+msgstr ""
+"Dies scheint das erste Mal zu sein das Sie LOOT v%s gestartet haben. "
+"Hier sind einige Tipps die Ihnen helfen "
+"Sie mit der Benutzeroberfläche vertraut zu machen"
+
+#: src/gui/html/js/translateStaticText.js:614
+msgid "Click %(menu_icon)s buttons to open menus."
+msgstr "Klicke die %(menu_icon) Schaltfläche um Menüs zu öffnen "
```

Comment:
This needs to preserve the whole of `%(menu_icon)s`, the trailing s is important.

Reply:
Figures that something slips by